server/sonar-server/src/main/java/org/sonar/server/permission/ws/GroupsAction.java

```diff
@@ -86,8 +86,8 @@ public void define(WebService.NewController context) {
   @Override
   public void handle(Request request, Response response) throws Exception {
     try (DbSession dbSession = dbClient.openSession(false)) {
-      Optional<ProjectRef> projectId = support.findProject(dbSession, request);
-      checkProjectAdminUserByComponentUuid(userSession, projectId.isPresent() ? projectId.get().getUuid() : null);
+      Optional<ProjectId> projectId = support.findProject(dbSession, request);
+      checkProjectAdmin(userSession, projectId);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "checkProjectAdmin"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)